editor/plugins/gradient_texture_2d_editor_plugin.cpp

```diff
@@ -33,81 +33,140 @@
 #include "editor/editor_node.h"
 #include "editor/editor_scale.h"
 #include "editor/editor_undo_redo_manager.h"
-#include "editor/gui/editor_spin_slider.h"
```

Comment:
Looks like you deleted it in a wrong file.

Reply:
Ah, true! I was a bit confused what you meant and was waiting for it to compile... now I see XD

Reply:
Though I don't see anything like this in the header, huh. I did do `class EditorSpinSlider` twice so I removed that tho.